tokio-util/src/stream/mod.rs

```diff
@@ -0,0 +1,117 @@
+//! Asynchronous value iteration.
+
+mod into_std;
+pub use into_std::IntoStd;
+
+use std::ops::DerefMut;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+/// A stream of values produced asynchronously.
+///
+/// This trait is used to convert Tokio's "stream-like" types into
+/// `futures::Stream`.  This trait is not intended to be implemented by third
+/// parties (use `futures::Stream` instead) and only exists to satisfy Rust's
```

Comment:
Since this isn't intended to be implemented by third parties, can/should it be kept private, sealed, or `#[doc(hidden)]`?